use_cases/eluc/predictors/sklearn/sklearn_predictor.py

```diff
@@ -83,21 +83,21 @@ def predict(self, X_test: pd.DataFrame) -> pd.DataFrame:
         :param X_test: DataFrame with input data
         :return: properly labeled DataFrame with predictions and matching index.
         """
-        if self.features:
-            X_test = X_test[self.features]
+        X_test = X_test[self.config["features"]]
         y_pred = self.model.predict(X_test)
-        return pd.DataFrame(y_pred, index=X_test.index, columns=[self.label])
+        return pd.DataFrame(y_pred, index=X_test.index, columns=[self.config["label"]])
 
 class LinearRegressionPredictor(SKLearnPredictor):
     """
     Simple linear regression predictor.
     See SKLearnPredictor for more details.
     """
     def __init__(self, model_config: dict):
+        if not model_config:
+            model_config = {}
         super().__init__(model_config)
-        model_config.pop("features", None)
-        model_config.pop("label", None)
-        self.model = LinearRegression(**model_config)
+        lr_config = {key: value for key, value in model_config.items() if key not in ["features", "label"]}
+        self.model = LinearRegression(**lr_config)
```

Comment:
Copy instead of referencing config so we don't remove features and label from our actual stored config